projects/angular/src/forms/styles/_textarea.clarity.scss

```diff
@@ -47,8 +45,19 @@
       forms-variables.$clr-forms-text-color,
       variables.$clr-use-custom-properties
     );
-    border-radius: forms-variables.$clr-forms-baseline * 0.5;
-    padding: $textarea-vertical-padding $textarea-horizontal-padding;
+    @include mixins.css-var(
+      border-radius,
+      clr-forms-textarea-border-radius,
+      forms-variables.$clr-forms-baseline * 0.5,
+      variables.$clr-use-custom-properties
+    );
+    @include mixins.css-var(
+      padding,
+      clr-forms-textarea-padding,
+      $textarea-vertical-padding $textarea-horizontal-padding,
+      variables.$clr-use-custom-properties
+    );
+
     font-size: forms-variables.$clr-forms-field-font-size;
 
     &:focus {
```

Comment:
Here you have to calculate selected status from figma using border color `--cds-alias-object-interaction-background-highlight`